packages/custom_lint_core/test/assist_test.dart

```diff
@@ -58,6 +64,36 @@ void main() {
     );
   });
 
+  test('CustomAssist.testRun', () async {
+    final assist1 = MyCustomAssist('CustomAssist', 'custom_1.txt');
+    final assist2 = MyCustomAssist('AnotherCustom', 'custom_2.txt');
+
+    final file = writeToTemporaryFile('''
+void main() {
+  print('Custom world');
+}
+''');
+    final result = await resolveFile2(path: file.path);
+    result as ResolvedUnitResult;
+
+    final changeList1 = await assist1.testRun(result, SourceRange.EMPTY);
+    final changeList2 = await assist2.testRun(result, SourceRange.EMPTY);
+
+    expect(
+      changeList1,
+      matcherNormalizedPrioritizedSourceChangeSnapshot('custom.json'),
```

Comment:
We don't care about normalized snapshots, since they don't include file paths – which is exactly what we care about here.

Reply:
Then I removed this expect() statement and the subsequent code and test files.